build/csharp_typemap.json

```diff
@@ -52,6 +52,7 @@
     "PROC": "nint",
     "COLORREF": "uint",
     "BOOL": "Silk.NET.Core.Bool32",
+    "bool": "Silk.NET.Core.Bool32",
```

Comment:
Have you regenerated all bindings to make sure this doesn't hit anywhere else? Feels like it'd be a common one!

Reply:
> Have you regenerated all bindings to make sure this doesn't hit anywhere else? Feels like it'd be a common one!

I dont think it should trigger anywhere else (webgpu is unique in that it uses all lowercase bool, which i dont think anyone else does, but ill regenerate rq)